src/frontend/src/pages/home/home.tsx

```diff
@@ -1,7 +1,94 @@
 import React from 'react';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import {
+    Link,
+} from "react-router-dom";
```

Comment:
```suggestion
import { Link } from "react-router-dom";
```
This can be on one line.